CODEOWNERS

```
@@ -319,6 +319,7 @@ Makefile* @cilium/build
 /cilium-cli/connectivity/tests/clustermesh-endpointslice-sync.go @cilium/sig-clustermesh
 /cilium-cli/connectivity/tests/egressgateway.go @cilium/egress-gateway
 /cilium-cli/connectivity/tests/encryption.go @cilium/sig-encryption
+/cilium-cli/connectivity/tests/encryption_v2.go @cilium/sig-encryption
```

Should we also add the `pod_to_pod_encryption_v2.go`?

```suggestion
/cilium-cli/connectivity/tests/encryption_v2.go @cilium/sig-encryption
/cilium-cli/connectivity/builder/pod_to_pod_encryption_v2.go @cilium/sig-encryption
```

---

hmm, is there  a reason the original `pod_to_pod_encryption.go` does not have a code owner? I was just copying what is currently there 🤔 

I dont see why your suggestion is not correct tho

---

Oh wait, nvm it is there in codeowners.